src/error.ts

```diff
@@ -23,7 +23,7 @@ export function createFetchError<T = any>(
   if (request && response) {
     message = `${message} (${response.status} ${
       response.statusText
-    } (${request.toString()}))`;
+    } (${JSON.stringify(request)}))`;
```

Comment:
Here, then we can have conditions to add request + additional info such as method from options